OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs

```diff
@@ -239,21 +239,26 @@ public async Task<List<ShortEntityDto>> GetWorkshopListByProviderAdminId(string
             providerAdminId);
 
         var providerAdmin = (await providerAdminRepository.GetByFilter(pa => pa.UserId == providerAdminId)).FirstOrDefault();
-        if (providerAdmin.IsDeputy)
+        if (providerAdmin != null)
```

Comment:
(Code readability) Please consider reverting this `if` condition and return empty list immediately when `providerAdmin` is `null`. This way you could decrease levels of code nesting.